spectacles/printer.py

```diff
@@ -93,11 +93,11 @@ def print_lookml_error(
 ) -> None:
     if file_path is None:
         file_path = "[File name not given by Looker]"
-    header_color = yellow if severity == "warning" else red
+    header_color = red if severity in ("fatal", "error") else yellow
```

Comment:
We don't need to account for info / success here, right?

Reply:
They'll just show up yellow. I tried blues but it didn't look right.